src/main/java/core/basesyntax/Square.java

```diff
@@ -0,0 +1,19 @@
+package core.basesyntax;
+
+class Square extends Figure {
```

Comment:
It's not clear whether the 'Figure' class is abstract or not. If 'Figure' is meant to define behavior without providing any implementation, it should be an interface instead of an abstract class.